crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -36,12 +41,49 @@ struct Fee(u128);
 const ACCOUNTS_PREFIX: &str = "accounts";
 const TRANSFER_BASE_FEE_STORAGE_KEY: &str = "transferfee";
 
-struct StorageKey<'a>(&'a Address);
-impl<'a> std::fmt::Display for StorageKey<'a> {
+trait GetAddressBytes: Send + Sync {
```

Comment:
I'd prefer if this were renamed `AddressBytes` and the method likewise, but not a big deal.

Reply:
Addressing this as part of #1318